### DISCUSSION TITLE
Allow remotes to add links to the same navigation section

### DISCUSSION THREAD

**Discussion by code-person** [gsoft-inc/wl-squide#146] [Ideas]
We have a pattern where multiple remotes need to add links to the same section. As of now, we need to create a specific menuId for this section. It would be more flexible if the API allowed adding a link to a section by id.

Also, teams currently split their route declarations by swimlane, supporting this would allow them to have more flexibility instead of relying on all navigation links to be declared at the same place.

![image](https://github.com/gsoft-inc/wl-squide/assets/77631430/ebeda505-48ac-4d47-9ec7-7a60ed99de74)

**Comment by patricklafrance** [comment]
Implemented in https://github.com/gsoft-inc/wl-squide/pull/204
